revocation/internal/crl/crl.go

```diff
@@ -153,14 +161,15 @@ func validate(crl *x509.RevocationList, issuer *x509.Certificate) error {
 
 	// check validity
 	now := time.Now()
+	// TODO IsZero return error
```

Comment:
This is required to be fixed in this PR.

Reply:
Updated.